examples/mixture/plot_bmm_means.py

```diff
@@ -0,0 +1,94 @@
+"""
+=================================
+Bernoulli Mixture Model Means
+=================================
+
+Plot the means of a mixture of Bernoulli distributions. This approximates
+the analysis of handwritten digits as described in Bishop[1], but uses
+synthetically generated data comprising X and + signs instead of digits.
+
+
+[1]
```

Comment:
Please use correct restructured text syntax for references or footnotes.

Reply:
updated.